### PR TITLE
cmark: add page

### PR DESCRIPTION
**Pull request by sbrl**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/cmark.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# cmark
+
+> Converts CommonMark Markdown formatted text to HTML
```
Comment (agnivade) [inline pages/common/cmark.md]: Since the tool converts to latex also, maybe we should say `.. formatted text to other formats` ?

Comment (pxgamer) [reply pages/common/cmark.md]: Sounds good to me. :+1:

Comment (sbrl) [reply pages/common/cmark.md]: Oh, oops! This mus  have slipped through the net. I'll update this now!

File: pages/common/cmark.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# cmark
+
+> Converts CommonMark Markdown formatted text to other formats.
+
+- Render a Commonmark Markdown file to HTML:
+
+`cmark --to html {{filename.md}}`
+
+- Render the standard input to latex:
```
Comment (agnivade) [inline pages/common/cmark.md]: How about "Convert data from standard input to latex" ?

Comment (sbrl) [reply pages/common/cmark.md]: Sure :-)

**Pull request merged**